content/doc/book/pipeline/getting-started.adoc

```diff
@@ -0,0 +1,226 @@
+
+* Click *New Item* on Jenkins home page.
+
+image::new-item-selection.png["Click *New Item* on the Jenkins home page", role=center]
+
+* Enter a name for your Pipeline, select *Pipeline* and click *OK*.
+
+image::new-item-creation.png["Enter a name, select *Pipeline*, and click *OK*", role=center]
+
+* In the *Script* text-area, enter a Pipeline script and click *Save*.
+
+image::hello-world-script.png["In the *Script* text-area, enter a Pipeline script and click Save", role=center]
+
+* Click *Build Now* to run the Pipeline.
+
+image::build-now.png["Click *Build Now* to run the Pipeline", role=center]
+
+
+* Click *#1* under "Build History" and then click *Console Output* to see the
+  full output from the Pipeline.
+
+image::hello-world-console-output.png["Click *Console Output* for the Pipeline", role=center]
+
+The example above shows a successful run of a basic Pipeline created in the Jenkins
+web UI, using two valuable steps.
+
+[pipeline]
+----
+// Script //
+node { // <1>
+    echo 'Hello World' // <2>
+}
+----
+<1> `node` to allocate an executor and workspace on the Jenkins cluster.
+<2> `echo` to echo a simple string in the Console Output.
+
+
+[CAUTION]
+====
+Jenkins uses the name of the Pipeline to create directories on disk. Pipeline
+names which include spaces may uncover bugs in scripts which do not expect
+paths to contain spaces.
+====
+
+=== Defining a Pipeline in SCM
+
+Complex pipelines would be cumbersome to write and maintain if you could only do
+that in the text area provided by the Jenkins job configuration page.
+
+Accordingly, you also have the option of writing pipeline scripts (Jenkinsfiles)
+with the editor that you use in your IDE (integrated development environment) or
+SCM system, and then loading those scripts into Jenkins using the *Pipeline
+Script from SCM* option enabled by the workflow-scm-step plugin, which is one of
+the plugins that the Pipeline plugin depends on and automatically installs.
+
+Loading pipeline scripts using the `checkout scm` step leverages the
+idea of "pipeline as code," and lets you maintain pipelines using version
+control and standalone Groovy editors.
+
+To do this, select *Pipeline script from SCM* when defining the pipeline.
+
+With the *Pipeline script from SCM* option selected, you do not enter any Groovy
+code in the Jenkins UI; you just indicate by specifying a path where in source
+code you want to retrieve the pipeline from. When you update the designated
+repository, a new build is triggered, as long as your job is configured with an
+SCM polling trigger.
+
+[TIP]
+====
+The first line of a `Jenkinsfile` should be `#!groovy`
+footnoteref:[shebang, https://en.wikipedia.org/wiki/Shebang_(Unix)]
+which text editors, IDEs, GitHub, etc will use to syntax highlight the
+`Jenkinsfile` properly as Groovy code.
+====
+
+
+== Built-in Documentation
+
+Pipeline ships with a number of built-in documentation features to make it
```

Comment:
ships with a number of built-in -> ships with built-in

There's only two mentioned below.